### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mgingras**

### CODE REVIEW

**Comment by QuinnWilton** [comment]
LGTM.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # edit-my-readme
 Edit my README. It's easy to start contributing!
+
+This is there you put important shit about the project but since there is no project this is kinda difficult.
```
Comment (ryanseys) [inline README.md:4]: I believe you meant to say "This is where*" :smile:

Comment (mgingras) [inline README.md:4]: Fuckin auto correct. Did that on my iphone.

**Comment by ryanseys** [comment]
Oops this became un-mergable as a result of merging #2. Can you rebase this very important contribution? :)

**Comment by mgingras** [comment]
Can I do that on my phone? ;P

Martin

**Comment by ryanseys** [comment]
@mgingras Not easily. The easiest way to do that from your phone that I can think of is to close this issue, re-fork the project, edit and re-create another PR. Not very efficient.

**Comment by ryanseys** [comment]
I did it for you :) Merged in 44e6dbbf4a4f1bc783c63d3b38807e815ddff12f

**Comment by mgingras** [comment]
:heart:  Lmao got on my computer to rebase it and it was in the history :+1:

**Comment by ryanseys** [comment]
:hamster:

**Pull request closed without merging**